mmv1/third_party/terraform/tests/data_source_google_vmwareengine_network_test.go.erb

```diff
@@ -0,0 +1,87 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceVmwareEngineNetwork_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"region":          acctest.GetTestRegionFromEnv(),
+		"random_suffix":   RandString(t, 10),
+		"organization":    acctest.GetTestOrgFromEnv(t),
+		"billing_account": acctest.GetTestBillingAccountFromEnv(t),
+	}
+
+	VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckVmwareengineNetworkDestroyProducer(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"random": {},
```

Comment:
It's not worth updating in this PR, but for future reference, I don't think you need this line because the "random" provider is not being used (that I could tell).

Reply:
Forgot to reply, but I had updated the code. Thank you